core/config/engine.h

```diff
@@ -201,6 +202,8 @@ class Engine {
 	bool notify_frame_server_synced();
 
 	void set_freeze_time_scale(bool p_frozen);
+	void set_embedded(bool p_enabled);
+	bool is_embedded() const;
```

Comment:
I worry that `Engine::is_embedded()` could be too vague. If/when we have LibGodot, that could be also thought of as "embedded". Should this maybe be `Engine::is_embedded_in_editor()` or something more specific?

Reply:
Done!